src/components/common/QuestionnaireLabel/NetPromoter/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { useCallback, useState } from 'react';
+import styles from './NetPromoter.module.css';
+import laborImage from './laborImage.png';
+
+const ScoreRange = () => {
+  const [score, setScore] = useState(5);
+  const ticks = Array.from({ length: 11 }, (_, i) => (
+    <div key={i} className="tick">
+      {i}
+    </div>
+  ));
+
+  return (
+    <div className={styles.range}>
+      <div className={styles.ticks}>{ticks}</div>
+      <input
+        type="range"
+        min={0}
+        max={10}
+        value={score}
+        step={1}
+        onChange={e => setScore(e.target.value)}
+      />
+      <div className={styles.rangeLabels}>
+        <span>完全沒幫助</span>
+        <span>非常有幫助</span>
+      </div>
+    </div>
+  );
+};
+
+const questionList = [
+  {
+    id: 1,
+    title: '你認為 GoodJob 網站對你找工作有幫助嗎？',
+    titleExplanation: null,
+    section: <ScoreRange />,
+  },
+  {
+    id: 2,
+    title: '你認為 GoodJob 有哪些地方可以做得更好嗎？',
+    titleExplanation: '(e.g.新增ＯＯ功能，優化ＸＸ功能)',
+    section: null,
+  },
+];
+
+const Question = ({ title, titleExplanation, section }) => {
+  return (
+    <React.Fragment>
+      <div className={styles.question}>
+        {title}
+        {titleExplanation && <span>{titleExplanation}</span>}
+      </div>
+      {section}
+    </React.Fragment>
+  );
+};
+
+const NextStepButton = ({ handleNext, isLastQuestion }) => {
+  const buttonText = isLastQuestion ? '送出' : '下一步';
+
+  const handleSubmit = useCallback(() => {
+    console.log('handleSubmit');
+  }, []);
+  const handleNextQuestion = useCallback(() => {
+    if (!isLastQuestion) {
+      handleNext();
+      return;
+    }
+    handleSubmit();
+  }, [handleNext, isLastQuestion, handleSubmit]);
```

Comment:
這邊可以考慮是否要將 submit 的邏輯不要放在 NextStepButton，而是在 NetPromoter

我的考量是：
1. NetPromoter 應該會維護一份表單的狀態 (state), 所以由它負責 submit
2. NextStepButton 看起來是個 UI，那傳入的 props 越直覺越好 (eg: instead of handleNext --> onClick)